src/internal/storage/track/tracker.go

```diff
@@ -12,8 +12,8 @@ import (
 )
 
 var (
-	// ErrObjectExists the object already exists
-	ErrObjectExists = errors.Errorf("object exists")
+	// ErrDifferentObjectExists the object already exists, with different downstream objects.
```

Comment:
Is this to handle the case where a file content chunk looks like an index chunk?

Reply:
No that is a separate issue to fix post GA.

We no longer error if the same object exists.  We check if the object has different references, and if it does, we error.  This eliminates a special case in the chunk package.